api/tests/lib/seed_local_db.py

```diff
@@ -142,13 +161,28 @@ def _build_agencies(db_session: db.Session) -> None:
     # Create a static set of agencies, only if they don't already exist
     agencies = db_session.query(Agency).all()
     agency_codes = set([a.agency_code for a in agencies])
+    agency_dict = {a.agency_code: a for a in agencies}
 
     for agency_to_create in AGENCIES_TO_CREATE:
         if agency_to_create["agency_code"] in agency_codes:
             continue
 
         logger.info("Creating agency %s in agency table", agency_to_create["agency_code"])
-        factories.AgencyFactory.create(**agency_to_create)
+
+        top_level_agency_code = agency_to_create.get("top_level_agency")
+        top_level_agency = None
+        if top_level_agency_code:
+            top_level_agency = agency_dict.get(top_level_agency_code)
+            if not top_level_agency:
+                logger.warning(
+                    f"Top-level agency {top_level_agency_code} not found for {agency_to_create['agency_code']}"
+                )
+
+        new_agency = factories.AgencyFactory.create(
+            agency_code=agency_to_create["agency_code"],
+            agency_name=agency_to_create["agency_name"],
+            top_level_agency=top_level_agency,
+        )
```

Comment:
Fine to just set the top level agency into the dict itself and expand it like before. Means we don't have to remember to pipe things through if we add other agency values to that config.

```suggestion
        agency_to_create["top_level_agency"] = top_level_agency
        factories.AgencyFactory.create(**agency_to_create)
```